### ISSUE TITLE
brace-style misses a pattern

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#644]
The `brace-style` rule misses this pattern, and it should be flagged:

``` js
if (foo) {

}
else {

}
```
